shell/platform/darwin/ios/BUILD.gn

```diff
@@ -182,7 +182,6 @@ source_set("flutter_framework_source") {
   ]
 
   frameworks = [
-    "AudioToolbox.framework",
     "CoreMedia.framework",
```

Comment:
If you want to pad you commit stats, I don't think we need CoreMedia either. The rest we do need.